src/extension.ts

```diff
@@ -216,6 +231,8 @@ export async function activate(context: vscode.ExtensionContext) {
         vscode.window.showInformationMessage(
           `Asset download successfully. Path: ${filePath}`
         )
+
+        outputChannel.appendLine(`Asset download successfully. Path: ${filePath}`)
       } catch (error) {
         console.error('Error details:', error)
```

Comment:
I think we should also append the errors cought within catch block.

Reply:
Thanks for the feedback, I've made changes to this.